eng/build.sh

```diff
@@ -403,6 +403,12 @@ while [[ $# > 0 ]]; do
       shift 1
       ;;
 
+     *crossbuild=true*)
+      crossBuild=1
+      extraargs="$extraargs $1"
+      shift 1
```

Comment:
Handled `/p:CrossBuild=true` this way to support multiple property cases like `./build.sh /p:'RuntimeFlavor=coreclr;CrossBuild=true;Configuration=release'`. Ideally, all cross-building CI legs should explicitly pass `-cross` (which appends `/p:CrossBuild=true`), but an explicit `/p:CrossBuild=true` should have the same effect.